docs/get-started/quickstart.md

```diff
@@ -53,7 +53,7 @@ mkdir multi_tool_agent/
 Now create an `__init__.py` file in the folder:
 
 ```shell
-echo "from . import agent" > multi_tool_agent/__init__.py
+echo "from . import agent\n\n__all__ = [\"agent\"]" > multi_tool_agent/__init__.py
```

Comment:
For better readability, can we split this line into 2-3 lines? (use echo multiple times)